libV2/config/defaults.json

```diff
@@ -41,7 +41,7 @@
         "schedule": "0 0 0 * * 6"
     },
     "diskUsage": {
-        "expirationBlockSize": "24",
-        "mode": "local"
+        "retentionPeriod": 45,
```

Comment:
could be useful to have the unit type appear in the tunable name (so `retentionDays`)